google/cloud/bigtable/ci/run_integration_tests_emulator_cmake.sh

```diff
@@ -32,6 +32,10 @@ readonly BINARY_DIR
 shift
 ctest_args=("$@")
 
+if ctest --test-dir "${BINARY_DIR}" --show-only -R "^bigtable_" "${ctest_args[@]}" 2>&1 | grep -q 'Total Tests: 0'; then
```

Comment:
Could we put a little `has_tests(test_dir, pattern)` helper for these in, say, `ci/cloudbuild/builds/lib/cmake.sh`?

Reply:
Good idea, done.

Reply:
Looks more like a "has_no_tests".

Reply:
Fixed.  Also split `ctest.sh` from `cmake.sh` because `cmake.sh` is too noisy.